filebeat/crawler/crawler.go

```diff
@@ -44,7 +44,7 @@ func (c *Crawler) Start(r *registrar.Registrar, reloaderConfig *common.Config) e
 	}
 
 	if reloaderConfig.Enabled() {
-		logp.Warn("EXPERIMENTAL feature dynamic configuration reloading is enabled.")
+		logp.Warn("BETA feature dynamic configuration reloading is enabled.")
```

Comment:
Me and Steffen discovered kind of a bug in the filebea reloading last Friday which will need some refactoring / cleanup to resolved. I'm not sure if we get this in until the release as it could touch quite a few parts. Do we have a place to mention such issues / known bugs? Just thinking if we perhaps should keep it experimental but support would probably be similar ...